components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -34,6 +34,11 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The color of the Tab bar's background. */
 @property(nullable, nonatomic, copy) UIColor *barTintColor;
 
+/**
+ The color for the Ripple effect for touch feedback. If no value is provided, uses a default value.
+ */
+@property(nullable, nonatomic, copy) UIColor *rippleColor;
```

Comment:
How do you feel about rippleColorForState?

Reply:
It doesn't support stateful ripple right now.  We can add that API later (and bind this to `.Normal` until deprecation/deletion).

Reply:
I spoke to @yarneo about it and it wasn't immediately obvious how it would work with the work @galiak11 is doing.  I think the simplest API is the best for our MVP.

Reply:
Curious to hear your feedback if you know better, though.

Reply:
That seems reasonable for now, please open a bug for this to track that we don't lose this.

Reply:
https://github.com/material-components/material-components-ios/issues/7786